### PR TITLE
Include additional notes for 3PAR multipath instructions (SCRD-8584)

### PR DESCRIPTION
**Pull request by cadenzajon**

### CODE REVIEW

File: xml/installation-installation-configure_3par.xml [hunk 1]
```diff
@@ -125,29 +125,23 @@
  <section xml:id="sec.3par-multipath">
   <title>Multipath Support</title>
   <para>
-   If you want to enable multipath for Cinder volumes carved from 3PAR FC/iSCSI
-   storage system, please go through the
+   Setting up multipath support is highly recommended for 3PAR FC/iSCSI
+   backends, and should be considered a default best practice.
```
Comment (asettle) [inline xml/installation-installation-configure_3par.xml]: "We recommend setting up multipath support for 3PAR FC/iSCSI backends as default best practice." just a rewording suggestion to cut it down a bit.

File: xml/installation-installation-configure_3par.xml [hunk 2]
```diff
@@ -125,29 +125,23 @@
...
+  <warning>
+   <para>
+    If multipath functionality is enabled, ensure that all 3PAR fibre channel ports are active and zoned correctly in the 3PAR storage for proper operation.
+   </para>
+  </warning>
   <para>
-   We have also included an additional set of steps needed if you are using
-   3PAR FC/iSCSI multipath which is included below.
+   In addition, take the following steps to enable 3PAR FC/iSCSI multipath
+   support in &ostack; configuration files:
```
Comment (asettle) [inline xml/installation-installation-configure_3par.xml]: "...support in the &ostack; configuration files:"

Comment (asettle) [reply xml/installation-installation-configure_3par.xml]: You could also probably remove "In addition"

File: xml/installation-installation-configure_3par.xml [hunk 3]
```diff
@@ -158,14 +152,19 @@
...
+     file and add this line under the <literal>[libvirt]</literal> section:
     </para>
     <para>
      Example:
     </para>
 <screen>[libvirt]
 ...
 iscsi_use_multipath=true</screen>
+    <para>
+     Additionally, if you are planning on attaching an encrypted 3PAR volume
```
Comment (asettle) [inline xml/installation-installation-configure_3par.xml]: Probably can remove "additionally" here too. Just "If you want to attach an encrypted 3PAR..."

File: xml/installation-installation-configure_3par.xml (hunk 1 above)
Comment (cadenzajon) [reply xml/installation-installation-configure_3par.xml]: Done

File: xml/installation-installation-configure_3par.xml (hunk 2 above)
Comment (cadenzajon) [reply xml/installation-installation-configure_3par.xml]: Not really, it needs to be very clearly a follow-on step associated with the previous clause.

Comment (cadenzajon) [reply xml/installation-installation-configure_3par.xml]: But I can use simpler wording and just change it to "also"

File: xml/installation-installation-configure_3par.xml (hunk 3 above)
Comment (cadenzajon) [reply xml/installation-installation-configure_3par.xml]: Done

**Pull request merged**